### DISCUSSION TITLE
Request for Implementation Function for Complete Cleanup of Blockscout Database.

### DISCUSSION THREAD

**Discussion by RaffaeleRusso1** [blockscout/blockscout#10893] [Q&A]
I am trying to implement a function in my application, which allows to completely clean up the Blockscout database, returning it to its initial state, as if it were the first use.

**Comment by ekawibisono** [comment]
Hello @RaffaeleRusso1 To completely clean up the Blockscout database and return it to its initial state, you can follow these steps:
If you are using Docker Compose, delete the redis-data, stats-db-data, and blockscout-db-data directories in the services folder [[2]](https://discord.com/channels/973915550965174292/1154430479467032646/1260479658072211528).
For a manual deployment, you can manually clean the instance by running ./rel/commands/clear_build.sh from the root project directory. This will delete the _build, deps, node modules, and logs directories. You should also delete the static assets in the apps/block_scout_web/priv/static folder [[1]](https://docs.blockscout.com/setup/deployment/manual-old-ui/check-your-manual-deployment-instance).
